core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseImpl.java

```diff
@@ -37,7 +38,7 @@
  * @since	7.0-B4
  *
  */
-public class LiquibaseImpl {
+public class LiquibaseImpl implements INamedObject{
```

Comment:
LiquibaseImpl is not really a named object, so this is a tricky/hacky way to pass the name.
I suggest you pass the Configuration as an argument to databaseMigrator.configure() in IbisContext line 349, and use that as the named object